GoogleSignIn/Sources/GIDGoogleUser_Private.h

```diff
@@ -27,6 +27,11 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithAuthState:(OIDAuthState *)authState
                       profileData:(nullable GIDProfileData *)profileData;
 
+// Update the granted scopes, auth state and profile data.
+- (void)updateScopes:(NSArray<NSString *> *)scopes
```

Comment:
No need to pass in scopes here as that value is contained within authState.

Reply:
Done.